config/webpack/webpack.common.js

```diff
@@ -20,6 +20,7 @@ const includeModules = [
     'react-native-modal',
     'react-native-onyx',
     'react-native-gesture-handler',
+    '@unimodules',
```

Comment:
Do we still need this?

Reply:
Just tried it without it, and it builds fine on web, so I'll get rid of it.